src/main/java/ts/parking/service/CashCalculations.java

```diff
@@ -0,0 +1,78 @@
+package ts.parking.service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+import ts.parking.model.Currency;
+import ts.parking.model.DriverType;
+import ts.parking.model.Parking;
+
+public class CashCalculations {
+
+	public static BigDecimal calculateParkingPaymentAmount(LocalDateTime start, LocalDateTime end, DriverType driverType, Currency currency) {
+		BigDecimal paymentAmount = BigDecimal.ZERO;
+		int parkingTimeInMinutes = Math.toIntExact(ChronoUnit.MINUTES.between(start, end));
+		int parkingTimeEachStartedHour = 0;
+		if(parkingTimeInMinutes % 60 > 0) {
```

Comment:
Nie rozumiem tego ifa, mógłbyś przybliżyć?

Reply:
Jeśli reszta z dzielenia przez 60 czasu w minutach będzie większa od zera to znaczy, że w czasie parkingu powinna zostać kolejna rozpoczęta godzina.

Reply:
*powinna zostać uwzględniona